sfge/src/test/java/com/salesforce/rules/fls/apex/ReadFlsScenariosTest.java

```diff
@@ -447,8 +468,9 @@ public void testSafeDbQueryAsNewMapInit() {
     }
 
     /** NPSP test case from BGE_DataImportBatchEntry_CTRL#getOpportunitiesWithOppPayments */
-    @Test
-    public void testSafeSecurityEnforcedInOuterQuery() {
+    @CsvSource({"SECURITY_ENFORCED", "USER_MODE"})
+    @ParameterizedTest(name = "{displayName}: {0}")
+    public void testSafeWithModeClauseInOuterQuery(String mode) {
```

Comment:
Looks like this test was missed out in the cleansing operation where we removed references to real-life repos. Could you rename object names and field names to something generic?